deployments/sandbox/environment.py

```diff
@@ -100,10 +115,10 @@ def mksrc(project,
     mksrc('datarepo-dev-59d37b9a', 'hca_dev_946c5add47d1402a97bba5af97e8bce7__20210831_20210903', 149),
     mksrc('datarepo-dev-788c3b52', 'hca_dev_955dfc2ca8c64d04aa4d907610545d11__20210831_20210903', 13),
     mksrc('datarepo-dev-4b88b45b', 'hca_dev_962bd805eb894c54bad2008e497d1307__20210830_20210903', 28),
-    mksrc('datarepo-dev-02c59b72', 'hca_dev_99101928d9b14aafb759e97958ac7403__20210830_20210903', 1190, ma=1),
+    mksrc('datarepo-dev-02c59b72', 'hca_dev_99101928d9b14aafb759e97958ac7403__20210830_20210903', 1190, flags=1),
```

Comment:
Why would we hard-code the flag value? Stick with the precedent from the prod branch, prod2 deployment.

Reply:
For PL: https://github.com/DataBiosphere/azul/blob/221b4c1d5c0c102335d0afd6e290d31cde16a5b0/deployments/prod2/environment.py#L23